change/@fluentui-react-positioning-29f6eb38-4af2-40f1-a095-6abef76d64f4.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30179&commitId=f4f436c6c581f19014419d69f55979faad7cd29a&env=prod&clientType=FLUENTUI)


  <details><summary><b>Positioning</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Positioning.auto size with async content.floating element is within the boundary.chromium.png|3434| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>

Reply:
The autosize diff is actually OK - it's equivalent to updating that test and calling `updatePosition` after the async content has loaded